challenge.rb

```diff
@@ -5,11 +5,22 @@
 # Complete the method `is_of_age?`. It should accept either strings or numbers in its arguments,
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
-def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+def is_of_age?(age, minimum_age)
+  if age.instance_of? Fixnum
+    if minimum_age.instance_of? Fixnum
+      age >= minimum_age
+    else
+      age >= minimum_age.to_i
+    end
+  else
+    if minimum_age.instance_of? Fixnum
+      age.to_i >= minimum_age
+    else
+      age.to_i >= minimum_age.to_i
+    end
+  end
```

Comment:
Hoo boy. 
Don't overthink this one. It's not a trick question - just `age >= minimum_age`.